app/serializers/api/v1/npq_application_csv_serializer.rb

```diff
@@ -82,6 +84,8 @@ def to_row(record)
           teacher_catchment(record),
           record.teacher_catchment_country,
           record.teacher_catchment_iso_country_code,
+          record.itt_provider,
```

Comment:
could you add a test for those in the `NPQApplicationCsvSerializer` spec for v1/v2? There seems to be a lot of duplication in the API specs, which hopefully we'll get to tidy up soon